designer/server/src/models/create.js

```diff
@@ -22,6 +24,21 @@ export function titleViewModel(metadata, validation) {
   }
 }
 
+/**
+ * Text and Value Item
+ * @typedef {object} Item
+ * @property {string} text - The item text
+ * @property {string} value - The item value
+ */
+
+/**
+ * @type {Item[]}
+ */
+const organisationItems = organisations.map((organisation) => ({
```

Comment:
I tried all sorts to remove the squiggles. Seemed like it should be easy. Please help!